.travis.yml

```diff
@@ -4,4 +4,5 @@ node_js:
 install:
   - yarn run init
 script:
-  - yarn run test
+  - yarn run lint
```

Comment:
`lint` should be included in another pull request.

Reply:
I'll send `lint` in the coming pull request. 👍

Reply:
The `lint` pull request is blocked by this pull request.